vendor/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go

```diff
@@ -562,7 +562,6 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 		}()
 
 		opts.Stdin = r
-		defer wg.Wait()
```

Comment:
Removing this is definitely correct. Normally there's a single `defer wg.Done()` at the very top of the spawned goroutine